### PR TITLE
Fixes #32 for deploying an app

### LINKED ISSUE #32: Create article on deploying a Blockstack Web App
- Explain how to deploy
- Include examples on common sites
- Include CORS samples

### PR DESCRIPTION
**Pull request by moxiegirl**
Signed-off-by: Mary Anthony <mary@blockstack.com>

### CODE REVIEW

**Comment by moxiegirl** [comment]
@reedrosenbluth Can you review this content? Yes, it is already merged, but I will make any changes you recommend.  

https://docs.blockstack.org/develop/deploy-tips.html

File: _develop/deploy-tips.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+layout: learn
+description: Blockstack app mining documentation
+permalink: /:collection/:path.html
+---
+# How to deploy a Blockstack dApp
+{:.no_toc}
+
+Blockstack applications are a type of web application that uses Blockstack's technology to authenticate users. As such, Blockstack apps tend to be simple in design and operation, since in many cases, they don’t have to host anything besides the application’s assets.
```
Comment (reedrosenbluth) [inline _develop/deploy-tips.md]: perhaps phrase this as: "Blockstack applications are web applications that authenticate users with _Blockstack Auth_ and store data with Gaia. Both of these technologies can be accessed on the client side. As such..."

File: _develop/deploy-tips.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    <td><code>cors</code></td>
+    <td>Contains example deployment files for cross-origin request configuration.</td>
+  </tr>
+</table>
+
+If you use the generator to build Javascript or Vue scaffolding, your project configuration files will be different. 
+
+Regardless of which scaffolding you use, you must customize and extend this basic scaffolding as needed by your application. For example, you may want to add more properties to the `manifest.json` file. Since every application is different, Blockstack cannot give you specific instructions on how to do this. The steps you take are specific to your application.
+
+## Blockstack Authentication and deployment
```
Comment (reedrosenbluth) [inline _develop/deploy-tips.md]: I think this section should mention that the included configs in the `cors` directory are specifically for making the `manifest.json` file accessible to the outside world (i.e. to the blockstack browser). If users are using something other than Netlify or Firebase, they will specifically need to configure CORS on their service of choice to include the  following headers when serving `manifest.json`:

```
Access-Control-Allow-Origin: *
Access-Control-Allow-Headers: "X-Requested-With, Content-Type, Origin, Authorization, Accept, Client-Security-Token, Accept-Encoding"
Access-Control-Allow-Methods: "POST, GET, OPTIONS, DELETE, PUT"```

Comment (moxiegirl) [reply _develop/deploy-tips.md]: done. Thanks for the review! @reedrosenbluth

**Pull request merged**